app/models/organization.rb

```diff
@@ -3,11 +3,13 @@ class Organization < ActiveRecord::Base
   has_many :users, through: :organizers
   has_many :items
 
-  validates :name, presence: true
-  validates :address, presence: true
-  validates :city, presence: true
+  validates :name, presence: true, length: { maximum: 250 }
+  validates :address, presence: true, length: { maximum: 250 }
+  validates :city, presence: true, length: { maximum: 100 }
   validates :zip, presence: true
-  validates :email, presence: true
+  validates_format_of :email, with: /@/
```

Comment:
Prefer the new style validations `validates :column, format: value` over `validates_format_of`.